llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -448,6 +454,142 @@ def construct_blocks(self, config: MPTConfig) -> nn.ModuleList:
             ) for _ in range(config.n_layers)
         ])
 
+    def _construct_blocks_with_overrides(
+        self,
+        config: MPTConfig,
+        block_args: Dict[str, Any],
+    ) -> nn.ModuleList:
+        if config.block_overrides is None:
+            raise ValueError(
+                'config.block_overrides should not be None when calling _construct_blocks_with_overrides.',
+            )
+        modules_order_expanded = {}
+        for type in 'start', 'repeating_pattern', 'end':
```

Comment:
i didn't know you could this...but maybe wrap in a list?

Reply:
Done.